tox.ini

```diff
@@ -0,0 +1,18 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```

Comment:
Travis automatically tests Python 2.7, 3.4, and 3.5, so using tox isn't necessary.